monitoring_201.rst

```diff
@@ -583,6 +583,17 @@ GDash
 Graphite.  Dashboards are created using a simple `DSL
 <https://github.com/ripienaar/graphite-graph-dsl/wiki>`_.
 
+BigPanda
+-----
```

Comment:
If I add more dashes until the end of BigPanda, will that be okay? Can I recommit?